package.json

```diff
@@ -56,13 +56,13 @@
     "prettier": "^3.3.3",
     "prettier-plugin-tailwindcss": "^0.6.8",
     "tailwindcss": "^3.4.14",
-    "typescript": "~5.6.2",
+    "typescript": "~5.9.3",
     "typescript-eslint": "^8.11.0",
     "vite": "^5.4.10",
     "vitest": "^2.1.4"
   },
   "volta": {
     "node": "22.11.0",
-    "pnpm": "9.12.3"
+    "pnpm": "10.24.0"
```

Comment:
Does pnpm pinning works with volta?
If not the please add instruction to use it in readme